### ISSUE TITLE
PSR12.Files.FileHeader conflicts with Generic.Files.LineEndings

### ISSUE THREAD

**Issue by on2** [squizlabs/PHP_CodeSniffer#2878]
If I run `phpcbf --sniffs="PSR12.Files.FileHeader,Generic.Files.LineEndings" myfile.php` where `myfile.php` is as follows (with `\r\n` line endings):

```php
<?php $name = 'Foo';

```

Then I receive an error:

> PHPCBF FAILED TO FIX 1 FILE

The file now contains 25 empty lines betwen the opening `<?php` tag and `$name = 'Foo';`. With `\r` or`\n` line endings the file is fixed as expected:

```php
<?php 

$name = 'Foo';

```

Tested with `dev-master` and v3.5.4.

**Comment by gsherwood** [comment]
Thanks for the report. This will be fixed in 3.5.5
